packages/ra-ui-materialui/src/makeOverridesResolver.tsx

```diff
@@ -0,0 +1,8 @@
+export function makeOverridesResolver(classes: { [key: string]: string }) {
+    return (props, styles) => [
+        styles.root,
+        ...Object.entries(classes).map(([name, cssClassName]) => ({
+            [`& .${cssClassName}`]: styles[name],
```

Comment:
This is not always right. Sometimes, the CSS class is applied on the element directly: `&.${cssClassName}` (note the space has been removed)